plugin/file/metrics.go

```diff
@@ -0,0 +1,53 @@
+package file
+
+import (
+	"github.com/coredns/coredns/plugin"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics for the secondary plugin
+var (
+	UpdatedSeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
that would lead to an import cycle becuase then `file` will import `secondary` to be able to use the metrics and `secondary`  already imports `file`

I have discussed this with you before [here](https://github.com/coredns/coredns/pull/2550#issuecomment-462134982) and I think you had agreed to it if I understood it correctly